chia/util/setproctitle.py

```diff
@@ -9,3 +9,11 @@
 def setproctitle(ps_name: str) -> None:
     if no_setproctitle is False:
         pysetproctitle.setproctitle(ps_name)
+
+
+def getproctitle() -> str:
+    if no_setproctitle is False:
+        # TODO: add type hints to setproctitle
+        return pysetproctitle.getproctitle()  # type: ignore[no-any-return]
+
+    return ""
```

Comment:
I suppose in this case, `setproctitle()` isn't likely doing anything anyway, right? (otherwise it might be nice to leave *some* clue in the name, like "chia")

Reply:
Yeah, this is the 'we do not have the ability to set titles' case.  So, this string only matters towards the end of being able to be combined with other strings right now.  If at some point we decided to print our title out the CLI or something...  Well, I might just say that we should have a name we manage and put to the CLI and the title instead of reading out of the title.